frontend/src/features/Driver/rideoffer.js

```diff
@@ -0,0 +1,86 @@
+import React, { Component } from 'react';
+import './driver.css';
+import { Link } from 'react-router-dom';
+
+class Rideoffers extends Component {
+    render() {
```

Comment:
This component doesn't have lifecycle methods you can change it to functional/ stateless component